### ISSUE TITLE
Look into Documentation around RAIL

### ISSUE THREAD

**Issue by dainemawer** [WordPress/performance#125]
As with other Handbooks in the WordPress eco-system - I foresee some of the work we're doing in the plugin, not only about optimising code and core, but also providing well-documented, insightful and pertinent documentation around "what makes a site performant" - after all, Core Web Vitals was created to help non-technical users get a better understanding of performance (without the technical know how)

My proposal is that we help users understand what makes a performant site. There is plenty of writing out there, but one that sticks out is: [RAIL](https://web.dev/rail/). This is a lens that web performance can be looked through, and I guess leans on the opinionated side of things, though, thats not too far detached from WordPress' ethos in general. 

The action item here would be to create a handbook that goes along with the Performance plugin, providing users with valuable insight into metrics, how things are measured, red flags to look out for an and other information that can help them better performance (i.e better plugin choices) but also better performance using the knowledge and wisdom of the WordPress community.

**Comment by manuelRod** [comment]
I agree! But doesn't this fall more into the documentation side of things? Like this [article](https://wordpress.org/support/article/optimization/) ?
I agree that more information and documentation should be basic for the community and a good starting point. 
After that, it would totally make sense to expand Site Health to track performance issues that can be solved, either automatically or by pointing them to good documentation, like [this site health module](https://github.com/WordPress/performance/issues/92).

**Comment by dainemawer** [comment]
@manuelRod I think you're right yes. Maybe we can start a Documentation focus @eclarke1 @felixarntz ?

**Comment by felixarntz** [comment]
@dainemawer I think this would make sense under the "Measurement" focus area maybe? We have "Documentation" as a type of issue, and I'm not sure it would make sense as a focus area, given that any focus area needs some documentation.

Is this issue more suitable in the Site Health focus or the Measurement focus? From the conversation so far, I'm leaning towards the latter.

**Comment by dainemawer** [comment]
@felixarntz awesome - I think lets tag it under Measurement. Will update

**Comment by adamsilverstein** [comment]
> The action item here would be to create a handbook that goes along with the Performance plugin

Great idea! If we see this as developer focused documentation, it would be good to consider how it might get merged into the existing theme developers handbook. If it is more user focused, it would be good to think about how it could eventually be part of an existing core help handbook or site. Maybe worth bringing up at a documentation team meeting.

**Comment by dainemawer** [comment]
Thanks for the insight @adamsilverstein ! Let me give the two paths some thought - I feel like there could be some cross over, especially if we're aligning with Googles vision of Page Experience Signals

I will chat to @eclarke1 about this - always enjoyed a bit of technical writing!

**Comment by soulgalore** [comment]
Just want to share that you be careful about the numbers in RAILs, you can read https://calendar.perfplanet.com/2018/magic-numbers/

**Comment by dainemawer** [comment]
Will give this a read thanks @soulgalore

**Comment by dainemawer** [comment]
@soulgalore I took a read through this - the authors opinion is sound, I think he makes an interesting argument and hope his own research proves the old literature wrong / misaligned. That being said - I don't think his opinion alone would convince me to drop a RAIL methodology on the basis that `0.1` may be incorrect or outdated, or more than likely out of range. If his results came back as even double that, say `0.2`, is it actually going to make a perceivable difference to a user? 

I think this ticket is less about the numbers and more about guidelines, it provides an outline for the pillars of performance and its life cycle. A 100 factors could effect any or all of the 4 pillars, and thats not really our problem to solve. It's more to bring context / knowledge into the space. 

Thanks for the link though - nice to see the flip-side of the coin

**Comment by swissspidy** [comment]
> The action item here would be to create a handbook that goes along with the Performance plugin, providing users with valuable insight into metrics, how things are measured, red flags to look out for an and other information that can help them better performance (i.e better plugin choices) but also better performance using the knowledge and wisdom of the WordPress community.

I don't think we have capacity for this. The core performance team so far has been really focused on moving the needle at scale through code changes. Writing such a handbook would be a huge lift. There's already tons of existing more general performance documentation out there too. And not to mention the big AI shift that flips this on its head as well.

I suggest closing this for now.
